.gitignore

```diff
@@ -22,6 +22,7 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+venv/
```

Comment:
We have just added `venv/` to .gitignore. So, we could revert this change

Reply:
OK: https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/ebb63d27ca7bbabd044553f68e3606bd43cbb426